docs/usage.md

```diff
@@ -16,7 +16,7 @@ Sarek is designed to handle single samples, such as single-normal or single-tumo
 The typical command for running the pipeline is as follows:
 
 ```console
-nextflow run nf-core/sarek --input samplesheet.csv  --outdir <OUTDIR> --genome GRCh38 -profile docker
+nextflow run nf-core/sarek --input samplesheet.csv  --outdir <OUTDIR> -profile docker
 ```
 
 This will launch the pipeline with the `docker` configuration profile. See below for more information about profiles.
```

Comment:
Just wanted to mention it's producing a `.nextflow.log` not `.nextflow_log` below

Reply:
And i think it should be **Sample sheet**